docs/pages/access-controls/access-monitoring.mdx

```diff
@@ -0,0 +1,428 @@
+---
+title: Getting Started with Access Monitoring
+description: Learn how to use Access Monitoring.
+---
+
+Access Monitoring is a powerful tool to understand and analyze the access patterns in your Teleport cluster.
+This guide will help you understand how to use Access Monitoring to monitor access to resources in your Teleport cluster.
```

Comment:
I would remove this sentence because it basically repeats the previous one.

Instead, I think what's missing is a short description of what access monitoring actually is and what it allows you to do. We're jumping straight into talking about configuration without explaining _what_ it actually does and _how_ do you actually monitor and analyze access in your cluster. I think we should give a brief description to help users build a mental model of what it is and how it works.

Something like this:

```
When Access Monitoring is enabled in a cluster, Teleport runs a background service that collects
signals/data from different input sources, builds graphical reports highlighting different usage patterns
within the cluster and allows users to create automated response actions.

The primary signal source for Access Monitoring system currently is the Teleport audit log. Out of the box
Teleport includes a report that searches the audit log for events indicating risky usage patterns (such as SSH
or database sessions without MFA) and provides users with recommendations on suggested actions.

Users are able to write their own custom access monitoring queries by querying the audit log represented as
a number of virtual database views using SQL.
```